guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -77,18 +96,21 @@ To create a host with unattended provisioning, complete the following steps:
 * The *Name* from the *Host* tab becomes the *DNS name*.
 * {ProjectServer} automatically assigns an IP address for the new host.
 +
-. In the *MAC address* field, enter a MAC address for the host. This ensures the identification of the host during the PXE boot process.
-. Ensure that {ProjectServer} automatically selects the *Managed*, *Primary*, and *Provision* options for the first interface on the host. If not, select them.
+. Note that {ProjectServer} automatically selects the *Managed*, *Primary*, and *Provision* options for the first interface on the host.
+. In the *MAC address* field, enter a MAC address of the host's provisioning interface. This ensures the identification of the host during the PXE boot process.
+. Click on *Save* and close the interface edit dialog. To add another inteface, click *Add*. Only one interface can be selected as *Provision* and *Primary*.
```

Comment:
> automatically selects the *Managed*, *Primary*, and *Provision* options

This is typical - we document an observation. However what actually is managed, primary and provision NIC we don't explain. And it's quite important. I try to improve here,  maybe it is worth explaining in more detail somewhere else.

Reply:
Added: For more information about network interfaces, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.5/html-single/managing_hosts/index#Adding_Network_Interfaces[Adding network interfaces].